mulesoft/docs-site-ui
# [W-15141521] fix jp preview site gulp tasks

ref: [W-15141521](https://gus.lightning.force.com/a07EE00001l5dXqYAI)

how to test:

1. run `npm run preview-jp` to start the JP preview site. Open the preview site (localhost:8080) in the web browser
2. change something in any of the hbs files or javascript files to trigger a hot reload
3. refresh the page in the web browser. The page should remain JP site instead of EN site

also added studio pages to the preview site.
